hebiros_gazebo_plugin/plugin/hebiros_gazebo_plugin.cpp

```diff
@@ -1,7 +1,54 @@
 #include <hebiros_gazebo_plugin.h>
 
+// Checks the content of a string before the first "." at compile time.
+// Necessary because Gazebo only defines string version numbers.
+// Note: This requires -std=c++14 or higher to compile
+constexpr int GetGazeboVersion ( char const* string_ver )
```

Comment:
Formatting here looks a bit awkward. Could you remove the spaces between the parameter and parens and cuddle the brace?

(Sorry for being insufferable...)